blackbox/docs/sql/administration/privileges.txt

```diff
@@ -0,0 +1,147 @@
+.. highlight:: psql
+.. _administration-privileges:
+
+=======================
+Cluster Wide Privileges
+=======================
+
+.. note::
+
+    Privileges are only part of the CrateDB `Enterprise Edition`_.
+    When the CrateDB Enterprise Edition is disabled, there will be
+    no user privilege checks, and every statement will be executed
+    without the validation of privileges.
+
+
+``Privilege Types``
+====================
+
+When a user is created, he doesn't have any privilege. The superuser can then
```

Comment:
this paragraph should be outside of this section since it has nothing to do with privilege types